cmd/utils/flags.go

```diff
@@ -591,6 +591,17 @@ var (
 		Value:    ethconfig.Defaults.Miner.NewPayloadTimeout,
 		Category: flags.MinerCategory,
 	}
+	MinerPriceCutoffPercentFlag = &cli.IntFlag{
+		Name: "miner.price_cutoff_percent",
+		Usage: "flashbots - The minimum effective gas price threshold used for bucketing transactions by price. " +
+			"For example if the top transaction in a list has an effective gas price of 1000 wei and price_cutoff_percent " +
+			"is 10 (i.e. 10%), then the minimum effective gas price included in the same bucket as the top transaction " +
+			"is (1000 * 10%) = 100 wei.\n" +
+			"NOTE: This flag is only used when miner.algotype=greedy-buckets",
+		Value:    -1,
```

Comment:
why not have this at `defaultPriceCutoffPercent`? having default as -1 is a bit misleading when using `geth --help` as that is not actually the default set in the code

Reply:
that's a really good point, updated to use `defaultPriceCutoffPercent`